README.md

```diff
@@ -243,10 +243,10 @@ TD, PID: 111924, SSH : ssh -p 10022 root@localhost
         ```console
         Id   Name                                                        State
         ---------------------------------------------------------------------------
-        1    tdvirsh-trust_domain-f7210c2b-2657-4f30-adf3-639b573ea39f   running (ssh:32855, cid:3)
+        1    tdvirsh-trust_domain-f7210c2b-2657-4f30-adf3-639b573ea39f   running (ip:192.168.122.212, hostfwd:32855, cid:3)
         ```
 
-        NOTE: `32855` in `ssh:32855` is the port number a user can use to connect to the TD via `ssh`.
+        NOTE: `32855` in `hostfwd:32855` is the port number a user can use to connect to the TD via `ssh`.
```

Comment:
I'm confused by this without looking at the rest (will is doing that). 

So, can we connect two ways now?

ssh 192.168.122.212
ssh 127.0.0.1 -p 32855

Or is it:
ssh 192.168.122.212 -p 32855

?

Reply:
We can connect in two ways, i kept the port forward method for backward comptability

Reply:
I would suggest to make it clear in that note in the readme.  Maybe providing two examples of how to run ssh or however you see best.

Reply:
Thanks for the suggestion but I think that for the sake of the README simplicity, we just need to offer one method, and the method we pick up to suggest is the one that has been here from the beginning and users are familiar with, for now, no need to recommend the other method nor advertise it, if in the future, if there is a need to change, we will do

Reply:
I think to @spmcmillan 's point I could still see this causing confusion. I would suggest explicitly showing example `ssh` commands that are supported.

Suggested change (I haven't tested these commands so please check):

NOTE: you can `ssh` to the guest in one of two ways: 

* Use the guest IP address directly:

```
ssh root@192.168.122.212
```

* Use a port on the host that is forwarded to the ssh port on the guest:

```
ssh -p 32855 root@localhost
```

Reply:
Sorry, I was writing that comment while you both were already discussing the same thing. :sweat_smile:

Reply:
Same answer than the one for @spmcmillan, i do not want to advertise this feature for now because it is for internal need,

Reply:
I'll go ahead and approve because I think it's your call but I think that because the new feature exposes new information to the user that it's reasonable to update the documentation to explain what that information means.

Reply:
@spmcmillan @frenchwr I finally made the change, i think if you both find it confusing, others might find it confusing as well. Please take a look if it works for you : https://github.com/canonical/tdx/pull/239/commits/3011929e8b1c1c04687a354f27c27326e00c2951

Reply:
Looks good! Thanks. :)